apps/web/src/components/features/groups/index.tsx

```
@@ -128,375 +129,438 @@ const mockGroups: Group[] = [
-            return 999
-          }
-          return getActivityScore(a.lastActivity) - getActivityScore(b.lastActivity)
+            if (activity.includes("minutes")) return 1000;
+            if (activity.includes("hour"))
+              return Number.parseInt(activity) || 100;
+            if (activity.includes("day"))
+              return (Number.parseInt(activity) || 1) * 24;
+            if (activity.includes("week"))
+              return (Number.parseInt(activity) || 1) * 168;
+            return 999;
+          };
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)